<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TATRC-17323" name="testCases.TeamLogin" tests="8" failures="2" timestamp="22 Oct 2019 22:36:58 GMT" time="154.869" errors="0">
  <testcase name="loginUser" time="19.042" classname="testCases.TeamLogin"/>
  <testcase name="loginUser" time="18.875" classname="testCases.TeamLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Page Title assertion:  expected [Team Fitness Tracker - Team Fitness Tracker] but found [Device Registration - Team Fitness Tracker]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Page Title assertion:  expected [Team Fitness Tracker - Team Fitness Tracker] but found [Device Registration - Team Fitness Tracker]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testCases.TeamLogin.loginUser(TeamLogin.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- loginUser -->
  <testcase name="loginUser" time="20.098" classname="testCases.TeamLogin"/>
  <testcase name="loginUser" time="19.520" classname="testCases.TeamLogin"/>
  <testcase name="loginUser" time="17.183" classname="testCases.TeamLogin">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Page Title assertion:  expected [Team Fitness Tracker - Team Fitness Tracker] but found [Device Registration - Team Fitness Tracker]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Page Title assertion:  expected [Team Fitness Tracker - Team Fitness Tracker] but found [Device Registration - Team Fitness Tracker]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testCases.TeamLogin.loginUser(TeamLogin.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- loginUser -->
  <testcase name="loginUser" time="19.157" classname="testCases.TeamLogin"/>
  <testcase name="loginUser" time="21.787" classname="testCases.TeamLogin"/>
  <testcase name="loginUser" time="19.207" classname="testCases.TeamLogin"/>
</testsuite> <!-- testCases.TeamLogin -->
